src/svs.js

```diff
@@ -59,16 +78,31 @@ function parseVSACXML(xmlString, vsDB = {}) {
   // Loop over the codes and build the JSON.
   const codeList = [];
   for (let concept in conceptList) {
-    let system;
-    const systemOID = conceptList[concept]['$']['codeSystem'];
-    if (typeof vsacCS[systemOID] !== 'undefined' && typeof vsacCS[systemOID].uri !== 'undefined') {
-      system = vsacCS[systemOID].uri;
+    let system = conceptList[concept]['$']['codeSystem'];
+    const code = conceptList[concept]['$']['code'];
+    const version = conceptList[concept]['$']['codeSystemVersion'];
+    const systemOid = `urn:oid:${system}`;
+    const systemUri = getVSACCodeSystem(vsacCS, system);
+
+    // Replace oid system with the url system, if one exists
+    if (options.svsCodeSystemType === 'url') {
+      if (systemUri !== null) {
+        system = systemUri.uri;
+      } else {
+        system = systemOid;
+      }
+    } else if (options.svsCodeSystemType === 'both') {
+      // Optionally include both if they exist
+      if (systemUri !== null) {
+        codeList.push({ code, system: systemUri.uri, version });
+      }
+      // Include the standard oid system
+      system = systemOid;
     } else {
-      system = `urn:oid:${systemOID}`;
+      // Keep the oid system as is
+      system = systemOid;
     }
```

Comment:
The "default" behavior here seems a little inconsistent. In my testing, I observed that:
* If no options are passed in, then it defaults to `{ svsCodeSystemType: 'url' }`
* If an _empty_ options is passes in (i.e., `{}`), then the logic falls through to the `else` and uses `oid`

To avoid confusion, I think empty options (or options without `svsCodeSystemType`) should work the same as the default when no options are passed in. And in order for this to be a non-breaking change, that default needs to be `url`. So this probably should be re-arranged to:
```
if (options.svsCodeSystemType === 'oid') {
  // use oid only
} else if (options.svsCodeSystemType === 'both') {
  // use oid and url (if available)
} else {
  // use url (if available, otherwise oid)
}
```